doc/setup.md

```diff
@@ -57,34 +101,28 @@ output:
 
 where:
 
-- `public_key[]` is an array of public keys generated by Signer[j] in Step 2.
-- `private_key[i]` is a private key of Signer[i] generated in Step 2.
-- `networkid` is Network ID we choose in Step 1. we use it as a nonce of encryption.
-- `block_height` is a 64-bits integer to be applied to this node vss. In initial setup, this should be 0. This option is optional, default is 0.
+- `public_key[]` is an array of public keys generated by Signer[j] in Step 1.
+- `private_key[i]` is a private key of Signer[i] generated in Step 1.
+- `networkid` is Network ID we choose. We use it as a nonce of encryption.
+- `block_height` is a 64-bits integer to be applied to this federation. In the initial setup, this should be 0. This option is optional, and the default is 0.
```

Comment:
`networkid` and `block_height` should be remove. it is removed from real command line arguments.

Reply:
removed